corehq/warehouse/management/commands/stage_table.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management import BaseCommand, CommandError
+from dimagi.utils.parsing import string_to_utc_datetime
+from corehq.warehouse.models import (
+    GroupStagingTable,
+)
+
+
+SLUG_TO_STAGING_TABLE = {
+    GroupStagingTable.slug: GroupStagingTable,
+}
+
+USAGE = """Usage: ./manage.py stage_table <slug> -s <start_datetime> -e <end_datetime>
+
+Slugs:
+
+{}
+
+""".format('\n'.join(sorted(SLUG_TO_STAGING_TABLE.keys())))
+
+
+class Command(BaseCommand):
+    """
+    Example: ./manage.py stage_table group_staging -s 2017-05-01 -e 2017-06-01
+    """
+    help = USAGE
+    args = '<slug>'
+
+    def add_arguments(self, parser):
+        parser.add_argument('slug')
+
+        parser.add_argument(
+            '-s',
+            '--start_datetime',
+            dest='start',
+            required=True,
+            help='Specifies the last modified datetime at which records should start being included',
+            type=_valid_date
+        )
+        parser.add_argument(
+            '-e',
+            '--end_datetime',
+            dest='end',
+            required=True,
+            help='Specifies the last modified datetime at which records should stop being included',
+            type=_valid_date
+        )
+
+    def handle(self, slug, **options):
+        start = options.get('start')
+        end = options.get('end')
+
+        model = SLUG_TO_STAGING_TABLE.get(slug)
+        model.stage_records(start, end)
```

Comment:
i forgot to mention this in our previous chats but it's a best practice any time we're moving data from one step to another to do a balancing step at the end where we make sure the count of records in the source matches the count of records in the destination, and if that doesn't match we should fail the process